chia/rpc/wallet_rpc_api.py

```diff
@@ -1881,7 +1878,7 @@ async def cancel_offers(self, request: Dict) -> EndpointResult:
         if asset_id is not None and asset_id != "xch":
             key = bytes32.from_hexstr(asset_id)
         while True:
-            records: List[TradeRecord] = []
+            records: Dict[bytes32, TradeRecord] = {}
```

Comment:
why not use a set?

Reply:
Because downstream in `cancel_pending_offers` we're going to want to look up a trade record by its ID and instead of making a redundant DB call, it has provided the option to pass in a dictionary for easy lookup.